Tests/Fuzzers/CMakeLists.txt

```diff
@@ -1,33 +1,3 @@
-cmake_minimum_required(VERSION 3.12)
-
-project(PcapPlusPlusFuzzer)
-
-set(CMAKE_PROJECT_HOMEPAGE_URL "https://pcapplusplus.github.io/")
-
-if(TARGET Pcap++)
-  # Are we building from the PcapPlusPlus project ??
-  message(STATUS "Building Fuzzer from PcapPlusPlus project")
-  # Alias the project to make it compatible with imported package
-  add_library(PcapPlusPlus::Pcap++ ALIAS Pcap++)
-else()
-  find_package(PcapPlusPlus CONFIG REQUIRED)
-  message(STATUS "Using Pcap++ ${PcapPlusPlus_VERSION}")
-  message(STATUS "Include dir: ${PcapPlusPlus_INCLUDE_DIR}")
-endif()
-
-# Set C++11
-set(CMAKE_CXX_STANDARD 11)
-# popen()/pclose() are not C++ standards
-set(CMAKE_CXX_EXTENSIONS ON)
-
 add_executable(Fuzzer FuzzTarget.cpp)
 
-target_link_libraries(Fuzzer PUBLIC PcapPlusPlus::Pcap++)
-
-# Compile and link with Fuzzer + ASAN
-target_compile_options(Fuzzer PRIVATE -g -fsanitize=fuzzer,address)
-target_link_options(
-  Fuzzer
-  PRIVATE
-  -g
-  -fsanitize=fuzzer,address)
+target_link_libraries(Fuzzer PUBLIC PcapPlusPlus::Pcap++)
```

Comment:
Not important but some of your text files does not have new line at the end. Since "txt" files are excluded from "pre-commit", it did not give any warning in ci.

Reply:
Yes good point.
Maybe we could add one or add cmake-format in the CI.

Reply:
thanks